thumbor/pyproject.toml

```diff
@@ -14,7 +14,15 @@ classifiers = [
 ]
 
 description = "Processes images enabling visulisation, resizing, reformating and retriving/storing of images in NAS S3 Buckets"
-dependencies = ["thumbor", "tornado", "boto3", image"]
+dependencies = [
+    "thumbor",
+    "tornado",
+    "image",
+    "pytz",
+    "pycurl",
+    "asyncio",
```

Comment:
Is `asyncio` needed as a dependency, or is it part of the standard library?

Reply:
It is part of the standard library - I will remove it.